bugwarrior/config.py

```diff
@@ -185,7 +185,12 @@ def get_taskrc_path(conf, main_section):
 
 
 def get_data_path():
-    return os.path.expanduser(os.getenv('TASKDATA', '~/.task'))
+    path = os.getenv('TASKDATA', '~/.task')
+    if conf.has_option(main_section, 'taskdata'):
+        path = conf.get(main_section, 'taskdata')
```

Comment:
Oh, missed that it's passed in as an argument above. Probably mis-copied :/ . Anyways, not necessary for `task _show`.